Source/SwiftLintFramework/Models/AccessControlLevel.swift

```diff
@@ -51,10 +51,10 @@ extension AccessControlLevel: Comparable {
     private var priority: Int {
         switch self {
         case .private: return 1
-        case .fileprivate: return 2
-        case .internal: return 3
-        case .public: return 4
-        case .open: return 5
+        case .fileprivate: return 1
+        case .internal: return 2
+        case .public: return 3
+        case .open: return 4
```

Comment:
this change might be a little too weird, in which case I can make this private to LowerACLThanBodyRule. For that use case private == fileprivate

Reply:
I added this Comparable conformance in my last PR, so I'm assuming this rule is still the only consumer, but I'm definitely happy to move it to be private

Reply:
I moved this into this rule's file